platforms/winit/src/lib.rs

```diff
@@ -171,70 +180,9 @@ impl Adapter {
         Self { adapter }
     }
 
-    #[cfg(not(all(
-        feature = "accesskit_unix",
-        any(
-            target_os = "linux",
-            target_os = "dragonfly",
-            target_os = "freebsd",
-            target_os = "netbsd",
-            target_os = "openbsd"
-        )
-    )))]
-    #[must_use]
-    pub fn on_event(&self, _window: &Window, _event: &WindowEvent) -> bool {
-        true
-    }
-    #[cfg(all(
-        feature = "accesskit_unix",
-        any(
-            target_os = "linux",
-            target_os = "dragonfly",
-            target_os = "freebsd",
-            target_os = "netbsd",
-            target_os = "openbsd"
-        )
-    ))]
     #[must_use]
     pub fn on_event(&self, window: &Window, event: &WindowEvent) -> bool {
```

Comment:
By now I think it is clear that we don't want to prevent the user from processing any winit event, so I think we should take this refactoring as an opportunity to remove `bool` from the return type of this function.

Reply:
I'm not sure. We definitely rejected the old way of handling keyboard input with AT-SPI, where we'd have to not only prevent the immediate handling of the event, but asynchronously re-inject the event later. Still, what if we later introduce a self-voicing adapter, basically an embedded screen reader, for environments that don't have a screen reader or accessibility API? Some of those environments might still use a port of winit. In that case, we'd want to prevent the application from handling some keypresses. What do you think?

Reply:
I wasn't planning this much into the future, but it could be useful indeed.